cds_dojson/marc21/fields/books/utils.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Document Server.
+# Copyright (C) 2017, 2018, 2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
```

Comment:
The copyright needs amendment: the years as well as the package name

Reply:
In first place I used the most recent short copyright description. But cds-dojson used GPL while the rest of invenio moved to MIT. Can you provide an updated version and I can replace them all.

Reply:
The license change is quite a big thing... You might go through in the Invenio sprint with files-rest, I would leave it for later the full license change...

Reply:
oh, I see it is everywhere like this. Ok then, to be tackled later on.

Reply:
@ludmilamarian do you thing the following would do the job?
```
# This file is part of CDS-DoJSON
# Copyright (C) 2015-2019 CERN.
#
# CDS-DoJSON is free software; you can redistribute it and/or
# modify it under the terms of the Revised BSD License; see LICENSE
# file for more details.
```

Reply:
I think yes, but Nicola is also right that this could be tackled later as part of a full license change.